services/optimization.js

```diff
@@ -108,6 +108,12 @@ Optimization.getOptimization = function(config) {
     });
   });
 
+  /**
+   * @typedef {Object} Distribution
+   * @property {number} pickup - indicates the index to the waypoint of the pick-up location
```

Comment:
Let's format these are sentences, with capitalization and punctuation.

What is this referencing the `index` of? Index within what array?

Reply:
Updated the PR, let me know if the wording is better.